packages/core/cli.ts

```diff
@@ -153,21 +154,15 @@ yargs(process.argv.slice(2))
 				sync: args.sync,
 				fuzzerOptions: args.corpus.concat(args._),
 				customHooks: args.custom_hooks.map((hookFile: string) => {
-					return path.join(process.cwd(), hookFile);
+					const cH = path.join(process.cwd(), hookFile);
+					return cH.endsWith(".js") ? cH : cH + ".js";
 				}),
 			};
-			if (args.sync) {
-				try {
-					startFuzzing(opts);
-				} catch (e: unknown) {
-					printError(e);
-				}
-			} else {
-				startFuzzingAsync(opts).catch((err) => {
-					printError(err);
-					stopFuzzingAsync();
-				});
-			}
+			const fuzzing = args.sync ? startFuzzing(opts) : startFuzzingAsync(opts);
+			Promise.resolve(fuzzing).catch((err: Error) => {
```

Comment:
`fuzzing` is already a promise, so no need to use `Promise.resolve`.

Reply:
True. Removed the call to `resolve`